src/osd/scrubber/scrub_job.h

```diff
@@ -375,6 +375,13 @@ class ScrubJob {
   static bool is_repair_implied(urgency_t urgency);
 
   static bool is_autorepair_allowed(urgency_t urgency);
+
+  /**
+   * should we cancel the repair if the number of damaged objects
+   * exceeds the configured limit ('osd_scrub_auto_repair_num_errors)?
```

Comment:
Not even tiny, rather piconit.

```suggestion
   * exceeds the configured limit ('osd_scrub_auto_repair_num_errors')?
```

Reply:
fixed. Thanks